department-of-veterans-affairs/caseflow
# MattT/APPEALS-9106: Change the Education RPO Completed Tab Description

Resolves [APPEALS-9106](https://vajira.max.gov/browse/APPEALS-9106)

### Description
The completed tabs for both Education Regional Processing Offices and the VHA Caregiver Support Program both filter the appeals that they list by the dates the respective organizations' tasks have been closed on. However, the description text on these two tabs differs slightly. For the sake of uniformity across all pre-docket workflows we would like to change the Education RPO completed tab to use the same copy as the one used in the VHA CSP completed tab.

### Acceptance Criteria
- [ ] The current description for the Completed Tab used by Education RPOs (Regional Processing Offices) reads "Cases completed in the last 7 days:". It should instead read "Cases completed (last 7 days):".

### Testing Plan
1. Log into Caseflow as an Education Regional Processing Office user (ex: EDURPOUSER)
2. Navigate to the organization queue for any of the three RPOs. Go to their Completed tab. ([/organizations/buffalo-rpo?tab=education_rpo_completed](http://localhost:3000/organizations/buffalo-rpo?tab=education_rpo_completed))
3. Ensure that the tab's description reads: "Cases completed (last 7 days):"


### User Facing Changes

 BEFORE|AFTER
 ---|---
![](https://i.imgur.com/KGb5a4w.png)|<img width="832" alt="edu-rpo-completed-tab-post-changes" src="https://user-images.githubusercontent.com/99351305/190003968-78eb2809-9bf4-4e5e-9ecf-cff23d3380ef.PNG">

### Code Coverage
<img width="405" alt="edu_rpo_completed_tab_coverage" src="https://user-images.githubusercontent.com/99351305/190005394-d798d5e2-fdee-40e2-b238-1d47de293ee6.PNG">
